src/WebJobs.Extensions.DurableTask/ActorScheduler/OperationMessage.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    internal class OperationMessage
+    {
+        [JsonProperty(PropertyName = "op")]
+        public string Operation { get; set; }
+
+        [JsonProperty(PropertyName = "signal")]
+        public bool IsSignal { get; set; }
+
+        [JsonProperty(PropertyName = "arg")]
+        public string Argument { get; set; }
```

Comment:
Have you considered other names, like `Data` or `Payload`? I wonder if those might be a bit more clear compared to `Argument`.

Reply:
Yes, I agree. We just need to agree on a better term. Data is perhaps a bit generic also. How about  `content`? This would be in analogy to the content of http requests.

Reply:
I like `Content` as well.